openapi.yaml

```diff
@@ -0,0 +1,315 @@
+    post:
+      summary: Create a new customer
+      security:
+        - BearerAuth: []
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - name
+                - email
+              properties:
+                name:
+                  type: string
+                email:
+                  type: string
+                segments:
+                  description: A list of segment labels to apply to the customer
+                  type: array
+                  items:
+                    type: string
+      responses:
+        201:
+          description: The customer was created successfully
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Customer"
+
+  /projects:
+    get:
+      summary: Get a list of projects
+      security:
+        - BearerAuth: []
+      responses:
+        200:
+          description: An array of project objects
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Projects"
+    post:
+      summary: Create a new project
+      security:
+        - BearerAuth: []
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - name
+              properties:
+                name:
+                  type: string
+                description:
+                  type: string
+      responses:
+        201:
+          description: The project was created successfully
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Project"
+
+    put:
+      summary: Update a project
+      security:
+        - BearerAuth: []
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - id
+              properties:
+                id:
+                  type: string
+                name:
+                  type: string
+                description:
+                  type: string
+      responses:
+        201:
+          description: The project was updated successfully
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Project"
+
+  /feedbackCategories:
+    get:
+      summary: Get a list of feedback categories
+      security:
+        - BearerAuth: []
+      responses:
+        200:
+          description: An array of category objects
+          content:
+            application/json:
+              schema:
+                properties:
+                  feedbackCategories:
+                    $ref: "#/components/schemas/FeedbackCategories"
```

Comment:
This was copied from the app repo, but updated to handle a few things better, clean things up, update them to the latest. Also while we have a create category API, I removed it from here as we don't really intend it to be used.